tree/ntuple/v7/inc/ROOT/RColumn.hxx

```diff
@@ -75,6 +75,13 @@ private:
    NTupleSize_t fFirstElementIndex = 0;
    /// Used to pack and unpack pages on writing/reading
    std::unique_ptr<RColumnElementBase> fElement;
+   /// The column team is a set of columns that serve the same column index for different representation IDs
+   /// Initially, the team has only one member, the very column it belongs to. Through MergeTeams(), two column
+   /// can join forces. The team is used to react on suppressed columns: if the current team member has a suppressed
+   /// column for a MapPage() call, it get the page from the active column in the corresponding cluster.
+   std::shared_ptr<std::vector<RColumn *>> fTeam;
+   /// Points into fTeam to the column that successfully returned the last page.
+   std::size_t fLastGoodTeamIdx = 0;
```

Comment:
This is to support reading pages from any of the (potentially suppressed) columns - do we need this? Shouldn't this be on the field to figure out which column to ask?

Reply:
As discussed, we may want to split the `RColumn` class in the future in a memory-facing part and the on-disk part, with the memory-facing side having potentially several on-disk backings. This may then make the logic clearer.